Lib/pdb.py

```diff
@@ -138,9 +138,12 @@ def find_function(funcname, filename):
 
             if funcdef:
                 try:
-                    funccode = compile(funcdef, filename, 'exec').co_consts[0]
+                    code = compile(funcdef, filename, 'exec')
                 except SyntaxError:
                     continue
+                # We should always be able to find the code object here
```

Comment:
Not sure the comment is needed

Reply:
Put it there as a soft assertion. If in the future somehow we failed to find the code object here, we know we had the wrong assumption before.